rpcs3/rpcs3qt/gs_frame.cpp

```diff
@@ -419,7 +419,7 @@ void gs_frame::flip(draw_context_t, bool /*skip_frame*/)
 	}
 }
 
-void gs_frame::take_screenshot(const std::vector<u8> sshot_data, const u32 sshot_width, const u32 sshot_height, bool is_bgra)
+void gs_frame::take_screenshot(const std::vector<u8>& sshot_data, const u32 sshot_width, const u32 sshot_height, bool is_bgra)
```

Comment:
this change should be wrong.
sshot_data is moved in line 530

Reply:
I didn't see it on line 530, fixed it.
[https://github.com/RPCS3/rpcs3/pull/9346/commits/a2dc10118be85840acb89a7e11c23af65a02a64b](url)